lib/ansible/modules/identity/ipa/ipa_host.py

```diff
@@ -37,6 +39,11 @@
     description:
     - Add the host to DNS with this IP address.
     required: false
+  host_otp:
+    description:
+    - OTP Password to add this host to IPA
+    required: false
+    version_added: "2.4"
```

Comment:
add an alias here for `userpassword`(to keep the consistency, I would even suggest to rename it to `user_password` because there is already a `user_` for `user_certificate`:
~~~diff
+    aliases: [ userpassword ]
~~~

Reply:
The actual API calls the field userpassword, so it feels appropriate to use that. I could put both aliases in if desired?

Reply:
I'm +1 for having an alias of `user_password`, keeping the convention used by other parameters.  With other parameters (example: mac_address), the API is `macaddress` but the parameter option is `mac_address`, making a bit more easily readable to users.